shims/spark310/src/main/scala/com/nvidia/spark/rapids/shims/spark310/ParquetCachedBatchSerializer.scala

```diff
@@ -122,7 +165,22 @@ private case class CloseableColumnBatchIterator(iter: Iterator[ColumnarBatch]) e
 class ParquetCachedBatchSerializer extends CachedBatchSerializer with Arm {
   override def supportsColumnarInput(schema: Seq[Attribute]): Boolean = true
 
-  override def supportsColumnarOutput(schema: StructType): Boolean = true
+  override def supportsColumnarOutput(schema: StructType): Boolean = schema.fields.forall(f =>
+    SQLConf.get.parquetVectorizedReaderEnabled && (isSupportedByCudf(f.dataType) ||
```

Comment:
So how is `SQLConf.get.parquetVectorizedReaderEnabled` going to interact with using this from the RAPIDS plugin?  If the user sets this config to false, which should only control the columnar reading _from the CPU_, do we really not want to read this columnar from the plugin?

If this config remains here, it should be moved outside of the `forall`.  It's not dependent on individual schema fields, and we could short-circuit the `forall` completely depending on the config setting.

Reply:
We shouldn't be reading that config. Thanks for bringing this up.